translate/cloud-client/translate_v3_detect_language.py

```diff
@@ -0,0 +1,45 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START translate_v3_detect_language]
+from google.cloud import translate
+
+
+def sample_detect_language(project_id="YOUR_PROJECT_ID"):
+    """Detecting the language of a text string."""
+
+    text = "Hello, world!"
```

Comment:
Should we inline this?

I think it should at least come after the `parent` line 26